src/main/fc/fc_rc.c

```diff
@@ -548,7 +552,27 @@ FAST_CODE void processRcCommand(void)
 #if defined(SIMULATOR_BUILD)
 #pragma GCC diagnostic pop
 #endif
-            calculateSetpointRate(axis);
+            bool useGpsRescueYaw = false;
```

Comment:
The flow control is already terrible and this is making it much worse ... there must be some cleaner way to handle this ...